CHANGELOG/v1.6.0.md

```diff
@@ -0,0 +1,294 @@
+  - [API deprecation details](https://main.cluster-api.sigs.k8s.io/contributing#removal-of-v1alpha3--v1alpha4-apiversions)
+- The API version `v1alpha3` has been completely removed in this release.
+- Flag `--metrics-bind-addr` is [deprecated](https://github.com/kubernetes-sigs/cluster-api/pull/9264) for all controllers
+
+## Changes since v1.5.0
+## :chart_with_upwards_trend: Overview
+- 388 new commits merged
+- 6 breaking changes :warning:
+- 15 feature additions ✨
+- 35 bugs fixed 🐛
+
+## :memo: Proposals
+- Community meeting: Add proposal for karpenter integration feature group (#9571)
+
+## :warning: Breaking Changes
+- API: Remove v1alpha3 API Version (#8997)
+- API: Stop serving v1alpha4 API Versions (#8996)
+- clusterctl: Improve Context handling in clusterctl (#8939)
+- Dependency: Bump to controller-runtime v0.16 (#8999)
+- Metrics/Logging: Implement secure diagnostics (metrics, pprof, log level changes) (#9264)
+- util: : Remove go-vcs dependency from releaselink tool (#9288)
+
+## :sparkles: New Features
+- API: Add validation to nested ObjectMeta fields (#8431)
+- CAPD: Add config maps to CAPD RBAC (#9528)
+- CAPD: Add MachinePool Machine implementation to CAPD components (#8842)
+- CAPD: Allow adding custom HA proxy config for CAPD load balancer (#8785)
+- CAPD: Initialize configmap object before getting it (#9529)
+- ClusterClass: Add topology-owned label to MachineHealthChecks. (#9191)
+- ClusterClass: Introduce NamingStrategy and allow generating names using go templates (#9340)
+- ClusterClass: Update API with ClusterClass MachinePool support (#8820)
+- clusterctl: Add RKE2 bootstrap provider to clusterctl (#9720)
+- clusterctl: Block move with annotation (#8690)
+- IPAM: Promote IPAM types to v1beta1 (#9525)
+- MachinePool: Add MachinePool workers support in ClusterClass (#9016)
+- MachineSet: Adjust preflight check to allow kubelet version skew of 3 for clusters running v1.28 and above (#9222)
+- Release: Add automation to create release branch and tags (#9111)
+- Testing/Documentation: V1.28: Prepare quickstart, capd and tests for the new release including kind bump (#9160)
+
+## :bug: Bug Fixes
+- CAPBK: Certificate paths in cloud-init scripts should not use a platform-dependent path separator (#9167)
+- CAPD: Delete container after failed start to work around port allocation issues (#9125)
+- CAPD: Fix DockerMachine panic (#9673)
+- CI: Fix reporting bug in verify-container-image script (#9676)
+- ClusterClass: Fix ClusterClass enqueue for ExtensionConfig (#9133)
+- ClusterClass: Topology: fix namingstrategy webhook to not use uppercase characters for testing the template and align unit test to e2e test (#9425)
+- clusterctl: Fix provider namespace secret not included in clusterctl move (#9694)
+- ClusterResourceSet: Requeue after 1 minute if ErrClusterLocked got hit (#9788)
+- Dependency: Bump golang.org/x/net to v0.13.0 (#9121)
+- Dependency: Bump to docker v24.0.5-0.20230714235725-36e9e796c6fc (#9038)
```

Comment:
Should we move these to others? Or are they actually bug fixes?

Reply:
It seems like this was fixing an issue: https://github.com/kubernetes-sigs/cluster-api/issues/9037

That said, i don't have a string opinion. It was your PR, your call :)

Reply:
Let's keep it as is :)